Gemfile

```diff
@@ -1,6 +1,9 @@
 source 'https://rubygems.org' do
-  group :development do
+  gem 'spoon' # Added because jruby seems to want it?
```

Comment:
Yeah, you are probably right! Hm....I need to think about this, then. I wonder if I should add it specifically to 'development' and 'test'. I don't want to require this in general. I'll test on Travis to see.